Marlin/Marlin_main.cpp

```diff
@@ -1713,6 +1718,26 @@ static void homeaxis(AxisEnum axis) {
       }
     #endif
 
+    #ifdef BABYSTEPPING
+    if(axis == Z_AXIS)
+    {
+      baby_max_endstop[axis] = Z_BABY_DEFAULT_MAX_POS;
+      baby_min_endstop[axis] = Z_BABY_DEFAULT_MIN_POS;
+    }
+    #ifdef BABYSTEP_XY
+    else if(axis == X_AXIS)
+    {
+      baby_max_endstop[axis] = X_BABY_DEFAULT_MAX_POS;
+      baby_min_endstop[axis] = X_BABY_DEFAULT_MIN_POS;
+    }
+    else if(axis == Y_AXIS)
+    {
+      baby_max_endstop[axis] = Y_BABY_DEFAULT_MAX_POS;
+      baby_min_endstop[axis] = Y_BABY_DEFAULT_MIN_POS;
+    }
+    #endif //BABYSTEP_XY
+    #endif //BABYSTEPPING
+
```

Comment:
> We already have a coding-standard and try to get the code conforming to it step by step.
> http://www.marlinfirmware.org/index.php/Coding_Standards <http://www.marlinfirmware.org/index.php/Coding_Standards>I’ve glad to see that you have a standard. My next question is “Is that a configuration file for `astyle` that matches this standard (particularly WRT indents and block delimiters)? If not, I suggest that you choose a standard that can be so described. That way, as a part of your Travis checking, you can also see if the style is followed and raise some alarm if it is not.

I would also like to see the standard specify that there be no whitespace at the end of lines.